src/main/java/net/glowstone/util/config/ServerConfig.java

```diff
@@ -650,7 +703,18 @@ public boolean test(String value) {
                 return false;
             }
             return true;
-        };
+        });
+
+        /**
+         * Creates a {@link Predicate} that checks if the value is an instance of the
+         * specified class.
+         *
+         * @param expected the expected class.
+         * @return the predicate
+         */
+        static <T> Predicate<T> typeCheck(Class<T> expected) {
+            return expected::isInstance;
```

Comment:
Inline this into lines 405-585, e.g. `DISABLE_GENERATION("world.disable-generation", false, Boolean::isInstance)`

Reply:
Wouldn't work in predicate chains, e.g. `Validators.typeCheck(Integer.class).and(Validators.ABSOLUTE)`

I'll replace it for single validators, i.e. `String.class::isInstance`